src/braket/circuits/braket_program_context.py

```diff
@@ -0,0 +1,82 @@
+from typing import List, Tuple
+
+import numpy as np
+
+from braket.circuits import Circuit, Instruction
+from braket.circuits.gates import Unitary
+from braket.circuits.translations import (
+    BRAKET_GATES,
+    braket_noise_gate_to_instruction,
+    braket_result_to_result_type,
+)
+from braket.default_simulator import KrausOperation
+from braket.default_simulator.openqasm.program_context import AbstractProgramContext
+from braket.ir.jaqcd.program_v1 import Results
+
+
+class BraketProgramContext(AbstractProgramContext):
+    def __init__(self):
+        super().__init__(Circuit())
```

Comment:
This no longer takes the circuit object

Reply:
How come we're inheriting from `AbstractProgramContext` and not `ProgramContext`? The latter has a `self.circuit` attribute and should take a circuit argument in its init method [see here](https://github.com/aws/amazon-braket-default-simulator-python/blob/main/src/braket/default_simulator/openqasm/program_context.py#L795C4-L795C4)

Reply:
Ok nvm about my question about the base class, I took another look and that makes sense now. I still thank any program context class with a `self.circuit` attribute should optionally take a circuit object as an init arg